src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -153,7 +153,7 @@ public OutsideRuntimeClient(ClientConfiguration cfg, bool secondary = false)
 
                 if (!secondary)
                 {
-                    UnobservedExceptionsHandlerClass.SetUnobservedExceptionHandler(UnhandledException);
+                    UnobservedExceptionsHandlerClass.TrySetUnobservedExceptionHandler(UnhandledException);
```

Comment:
Unfortunately that we still do this, especially on the client, where Orleans is not the runtime, just a small client library. We shouldn't be the ones hooking up logging for any unobserved exception that could occur in the frontend.
Nevertheless, is trying to set it and skip otherwise the right choice? Could there be side effects if I start clientA, then clientB, and then dispose clientA?